scripts/invite-collaborator

```diff
@@ -19,7 +19,7 @@ class Team
 
   def team
     @team ||= gh.organization_teams('cucumber')
-      .find { |team| team['name'] == name } || raise("Unable to find a team named #{name}")
+                .find { |team| team['name'] == name } || raise("Unable to find a team named #{name}")
```

Comment:
This is a preference (And it isn't in rubocop), but I'd start a newline after `||=` which means less tabbing

rubocop will not complain about :point_up:

Reply:
I see what you mean. I don't know if I agree here, but I'll take a look later. :) Ultimately the way it is now satisfies Rubocop which is the goal. Thanks for the input!

Reply:
Fixed it. :)